tla/ccfraft.tla

```diff
@@ -572,31 +572,34 @@ SignCommittableMessages(i) ==
 \* This will switch the current set of servers to the proposed set, ONCE BOTH
 \* sets of servers have committed this message (in the adjusted configuration
 \* this means waiting for the signature to be committed)
-ChangeConfiguration(i, newConfiguration) ==
-    \* Only leader can propose changes
-    /\ state[i] = Leader
-    \* Limit reconfigurations
-    /\ IsInConfigurations(i, newConfiguration)
-    \* Configuration is non empty
-    /\ newConfiguration /= {}
-    \* Configuration is a proper subset of the Servers
-    /\ newConfiguration \subseteq Servers
-    \* Configuration is not equal to the previous configuration
-    /\ newConfiguration /= MaxConfiguration(i)
-    \* Keep track of running reconfigurations to limit state space
-    /\ reconfigurationCount' = reconfigurationCount + 1
-    /\ removedFromConfiguration' = removedFromConfiguration \cup (CurrentConfiguration(i) \ newConfiguration)
-    /\ LET
-           entry == [term |-> currentTerm[i],
-                    value |-> newConfiguration,
-                    contentType |-> TypeReconfiguration]
-           newLog == Append(log[i], entry)
-           IN
-           /\ log' = [log EXCEPT ![i] = newLog]
-           /\ configurations' = [configurations EXCEPT ![i] = @ @@ Len(log[i]) + 1 :> newConfiguration]
-    /\ UNCHANGED <<messageVars, serverVars, candidateVars, clientRequests,
-                    leaderVars, commitIndex, committedLog>>
+ChangeConfigurationInt(i, newConfiguration) ==
```

Comment:
This change is just renaming and indention right? Just want to check I'm not missing anything

Reply:
It's alright, I just wondered if I was misreading the diffs